Add missing function declaration.
There was no header files. definition logic is copied over from where function was defined

author: micketeer@gmail.com
--- Source/LibJXR/image/decode/segdec.c.orig	2025-10-04 13:42:59.144365467 +0000
+++ Source/LibJXR/image/decode/segdec.c	2025-10-04 13:44:06.698371172 +0000
@@ -36,6 +36,17 @@
 #include "memtrace.h"
 #endif
 
+#if (defined(WIN32) && !defined(UNDER_CE) && (!defined(__MINGW32__) || defined(__MINGW64_TOOLCHAIN__))) || (defined(UNDER_CE) && defined(_ARM_))
+// WinCE ARM and Desktop x86
+#else
+// other platform
+#ifdef _BIG__ENDIAN_
+#define _byteswap_ulong(x)  (x)
+#else // _BIG__ENDIAN_
+U32 _byteswap_ulong(U32 bits);
+#endif // _BIG__ENDIAN_
+#endif
+
 extern const int dctIndex[3][16];
 extern const int blkOffset[16];
 extern const int blkOffsetUV[4];
